src/components/VideoPlayer/index.js

```diff
@@ -145,7 +145,12 @@ export default class VideoPlayer extends PureComponent {
       this.video.play()
     } else if (endscreenComponent) {
       this.setState({ endscreenOpen: true })
-    } else if (pausescreenComponent) {
+      // eslint-disable-next-line
+      if (this.video.isFullscreen_) {
```

Comment:
is there any downside to executing `closeFullscreen` if the video isn't in fullscreen?